catroid/src/main/java/org/catrobat/catroid/io/XstreamSerializer.java

```diff
@@ -460,6 +461,7 @@ private void prepareXstream(Class projectClass, Class sceneClass) {
 		xstream.alias("brick", SetBackgroundAndWaitBrick.class);
 		xstream.alias("brick", SetBackgroundByIndexAndWaitBrick.class);
 		xstream.alias("brick", SetInstrumentBrick.class);
+		xstream.alias("brick", PlayDrumForBeatsBrick.class);
```

Comment:
Same as discussed with @m-herold for normal instruments: please make a token based serialization, something like:
```
<brick type="SetInstrumentBrick">
  <commentedOut>false</commentedOut>
  <drumSelection>TRIANGLE</instrumentSelection>
</brick>
```
and so on.